### PR TITLE
9096 iOS Wrap options text

### PR DESCRIPTION
**Pull request by Sergio-Mira**
https://swrvedev.jira.com/browse/SWRVE-9096

### CODE REVIEW

**Comment by Sergio-Mira** [comment]
@dominicmarmionswrve Could you have a look? :bear:

File: Sdk/Conversation/SwrveInputMultiValue.m [hunk 1]
```diff
@@ -122,7 +128,28 @@ -(CGFloat) heightForRow:(NSUInteger) row {
...
+                                                       options:NSStringDrawingUsesLineFragmentOrigin
+                                                       context:nil];
+            height = ceil(rect.size.height);
+        } else {
+            CGSize labelSize = [cellText sizeWithFont:cellFont constrainedToSize:constraintSize lineBreakMode:NSLineBreakByWordWrapping];
+            height = ceil(labelSize.height);
+        }
+        
+        return height + 22;
     }
```
Comment (dominicmarmionswrve) [inline Sdk/Conversation/SwrveInputMultiValue.m:139]: Looks good! Could we use similar solution for header part (when row == 0)?

Comment (Sergio-Mira) [inline Sdk/Conversation/SwrveInputMultiValue.m]: Wop! Maybe sizeWithFont constrainedToSize works everywhere (even in iOS6 where the other method is deprecated)

**Comment by dominicmarmionswrve** [comment]
:rocket:

**Comment by Sergio-Mira** [comment]
@dominicmarmionswrve Updated to use the same method and approach for the header too

**Comment by dominicmarmionswrve** [comment]
Nice! :rocket:

**Pull request merged**